test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -131,14 +91,14 @@ public class ContentResultTest
                         new MediaTypeHeaderValue("text/foo"),
                         "κόσμε",
                         null,
-                        "text/foo; charset=utf-8",
+                        "text/foo",
```

Comment:
But this is not just refactoring though right...we have a behavior change as mentioned earlier...